### PR TITLE
Add ERC: Release ERC-20 tokens on predefined periods

### PR DESCRIPTION
**Pull request by erc4377**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-4377.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
+---
+eip:
```
Comment (abcoathup) [inline ERCS/erc-4377.md:2]:
```suggestion
eip: 7660
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename and all references to the ERC number.

File: ERCS/erc-4377.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+title: A Protocol for ERC20 Tokens Which Released Based On Predefined Periods Controlled by Smart Contracts
+description: Controlled by smart contracts, tokens can be linearly released based on the predefined period.
+author: Roy Davies(roydj2021@gmail.com), Ali Kaya (ali.kayatur747@gmail.com)
+discussions-to: https://ethereum-magicians.org/t/erc4377-a-protocol-for-erc20-tokens-which-released-based-on-predefined-periods-controlled/19305
```
Comment (abcoathup) [inline ERCS/erc-4377.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7660-a-protocol-for-erc20-tokens-which-released-based-on-predefined-periods-controlled/19305
```

File: ERCS/erc-4377.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
+---
+eip:
+title: A Protocol for ERC20 Tokens Which Released Based On Predefined Periods Controlled by Smart Contracts
```
Comment (abcoathup) [inline ERCS/erc-4377.md:3]: Suggest shortening the title, perhaps something like "Release ERC20 tokens on predefined periods"

**Comment by erc4377** [comment]
update eip Number and ERC-20

**Comment by erc4377** [comment]
shortening the title

**Comment by erc4377** [comment]
update format

**Comment by erc4377** [comment]
shortening the title update

**Comment by gaudren** [comment]
Did you mean to close this, or did you still want somebody to review it?

**Pull request closed without merging**